include/openenclave/internal/utils.h

```diff
@@ -228,6 +228,33 @@ OE_INLINE void oe_mem_reverse_inplace(void* mem, size_t n)
     }
 }
 
+OE_INLINE unsigned char oe_hex_digit_to_num(char c)
+{
+    if (c >= '0' && c <= '9')
+        return (unsigned char)(c - '0');
+
+    if (c >= 'A' && c <= 'F')
+        return (unsigned char)(10 + (c - 'A'));
+
+    return (unsigned char)(10 + (c - 'a'));
+}
+
+OE_INLINE oe_result_t
+oe_hex_to_buf(const char* str, size_t strsize, uint8_t* buf, size_t bufsize)
```

Comment:
OE style guidelines say to avoid abbreviations
```suggestion
oe_hex_to_buffer(const char* hex_string, size_t hex_string_size, uint8_t* buffer, size_t buffer_size)
```